src/OrchardCore.Modules/OrchardCore.Taxonomies/Controllers/AdminController.cs

```diff
@@ -284,6 +291,64 @@ public async Task<IActionResult> Delete(string taxonomyContentItemId, string tax
             return RedirectToAction("Edit", "Admin", new { area = "OrchardCore.Contents", contentItemId = taxonomyContentItemId });
         }
 
+        public async Task<IActionResult> OrderCategorizedContentItems(string taxonomyContentItemId, string taxonomyItemId)
```

Comment:
POST

Reply:
ok

Reply:
This is a GET, called when opening the page or when the pager is clicked. The POST is OrderCategorizedContentItemsPost, called when a categorized content item is moved to a new position.

Reply:
I saw an object being modified. But there is no Save so it's ok. However altering this object seems weird.

Reply:
On the last version of this method, I'm using the `TermPart` to pass some information to the `TermPartContentDriver` (Ordering and OrderingPageSize ). `TermPart` is never saved. I'm reusing the Driver for the ordering page, that's why I extract the TermPartViewModel from the result. This way, the Driver's `DisplayAsync` method is used for both the front end and the ordering page, instead of creating separate code for the ordering page.
Contrary to what happens on the Lists module, here we are dealing with content items (terms) that are stored inside a parent content item (taxonomy), witch forces less orthodox solutions.